README.md

```diff
@@ -34,20 +34,22 @@ NOTE: I am not going to write this for beginnings. I assume if you are smart eno
 ```bash
 make O=out ARCH=arm64 <defconfig>
 
-PATH="<path to clang folder>/bin:<path to gcc folder>/bin:${PATH}" \
+PATH="<path to clang folder>/bin:<path to gcc folder>/bin:<path to 32 bit gcc folder>/bin:${PATH}" \
```

Comment:
Mind clarifying `64-bit gcc folder` then?

Reply:
Do you mean: s/<path to gcc folder>/<path to 64 bit gcc folder>/ ???

Reply:
Yes!